clients/admin-ui/src/features/common/custom-fields/form/CustomInput.tsx

```diff
@@ -17,6 +18,21 @@ import { FieldHookConfig, useField } from "formik";
 
 import { InputType } from "./types";
 
+/**
+ * This isn't the cleanest way to style these forms, but it's a quick hack. Ideally our form inputs
+ * (like CustomSelect) would all use the same styles, or at least have a limited set of variants:
+ * https://chakra-ui.com/docs/styled-system/component-style#styling-multipart-components
+ *
+ * Unfortunately the designs for these forms call for styles that are inconsistent with the other
+ * forms in the app.
+ */
+export const CUSTOM_LABEL_STYLES: FormLabelProps = {
+  color: "gray.600",
+  fontSize: "14px",
+  fontWeight: "semibold",
+  minWidth: "150px",
+};
```

Comment:
[Post-review]: The whole component below should go away in favor of:
- Using input fields from `~/features/common/form/inputs`
- Adding the ability to override label styles ([for now](https://ethyca.slack.com/archives/C043URV1KLL/p1676061654400279?thread_ts=1676054836.626409&cid=C043URV1KLL))
- Extracting the useful parts of the number input to that module so we could use it elsewhere if needed.